docs/pages/script.js

```diff
@@ -0,0 +1,10 @@
+import { Jooa11y, Lang } from '../../dist/js/joomla-a11y-checker.standalone.esm.js';
+import EngSetting from '../../dist/js/lang/en.js';
+
+// Set translations
+Lang.addI18n(EngSetting.strings);
+
+// Instantiate
+const checker = new Jooa11y(EngSetting.options);
```

Comment:
Please use window load event, as it was in original code.
The scan should be run only when everything is loaded.

Reply:
Ok

Reply:
document is loaded but images are not, also the document not fully rendered

Reply:
Done

Reply:
thanks!